src/controllers/auth.js

```
@@ -37,6 +37,20 @@ const login = async (req, res) => {
 }
 
+const authGoogle = async (req, res) => {
+  const { token, role } = req.body
+  const lang = req.lang
+
+  const tokens = await authService.googleAuth(token.credential, role, lang)
+
+  res.cookie(ACCESS_TOKEN, tokens.accessToken, COOKIE_OPTIONS)
+  res.cookie(REFRESH_TOKEN, tokens.refreshToken, COOKIE_OPTIONS)
+
+  delete tokens.refreshToken
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)